requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3==1.12.23
+moto==1.3.16
+pytest==6.0.1
```

Comment:
this shouldn't be in the requirements file, maybe in a requirements_dev.txt instead?

Reply:
you mean the libraries used for testing? so leave boto3 in there but move the rest?